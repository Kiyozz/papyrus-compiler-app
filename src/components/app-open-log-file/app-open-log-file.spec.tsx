import { shallow } from 'enzyme'
import React from 'react'
import AppOpenLogFile from './app-open-log-file'

describe('<AppOpenLogFile>', () => {
  it('renders correctly', () => {
    const component = (
      <AppOpenLogFile openLogFile={() => {}} />
    )

    expect(component).toBeTruthy()
  })

  it('matches snapshot', () => {
    const component = shallow((
      <AppOpenLogFile openLogFile={() => {}} />
    ))

    expect(component).toMatchSnapshot()

    const button = component.find('button')

    expect(button).toExist()
    expect(button).toHaveProp('onClick')
  })
})
