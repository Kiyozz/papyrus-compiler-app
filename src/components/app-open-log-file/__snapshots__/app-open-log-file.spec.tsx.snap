// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppOpenLogFile> matches snapshot 1`] = `
<div
  className="app-open-log-file"
>
  <button
    className="btn btn-outline-secondary app-open-log-file-button-activate d-flex justify-content-center align-items-center"
    onClick={[Function]}
  >
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon="file"
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      symbol={false}
      title=""
      transform={null}
    />
  </button>
</div>
`;
